deploy/charts/istio-csr/values.yaml

```diff
@@ -19,6 +19,10 @@ app:
   # -- Verbosity of istio-csr logging.
   logLevel: 1 # 1-5
 
+  metrics:
+    # -- Port for exposing Prometheus metrics on 0.0.0.0 on path '/metrics'.
+    port: 9402
```

Comment:
Suggestion: either expose this port via the existing `cert-manager-istio-csr` `Service` or create a new `Service`, so it's easier to consume

Suggestion2: it might also be nice to optionally create a `ServiceMonitor` for the `Service`

Reply:
Thanks, I will add a separate Service and optional service monitor

Reply:
@irbekrm Added in https://github.com/cert-manager/istio-csr/pull/72/commits/b8641b0d291ae266dcd2f7f8f57e1cdae5b48044 and https://github.com/cert-manager/istio-csr/pull/72/commits/e92d05acae3970598869fe52c68f58d93a98db74